### ISSUE TITLE
Renders incorrect font

### ISSUE THREAD

**Issue by jamesdwork** [ant-design/ant-design#30684]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/basic-antd4160-forked-54qkg?file=/index.js)

### Steps to reproduce
1) Go to https://54qkg.csb.app/ in a Chrome based browser
2) Open DOM Inspector and view the font for the number

Alternatively
1) Go to an ant.design page where there is a number, eg. https://ant.design/components/input-number/ in a Chrome based browser
2) Open DOM Inspector and change the font for a number to Montserrat

In both cases the number renders differently from the antd default, but is not the Montserrat font (the number 1 in particular looks wrong). Strangely the inspector indicates it is. If you look on other websites (that do not use antd), they render the Montserrat font correctly. For the correct look of the font, see https://fonts.google.com/specimen/Montserrat?selection.family=Montserrat&preview.text=127&preview.text_type=custom

### What is expected?
It should show the actual font that is in the Inspector.

### What is actually happening?
Wrong font displayed and does not match the font that the inspector mentions

| Environment | Info |
|---|---|
| antd | 4.16.0 |
| React | 16.8 |
| System | Windows 10 |
| Browser | chrome |

---
The website I'm working on (not available to the public) used to render the correct font, but once we imported the antd libraries, it changed to the wrong font, even though the inspector indicates it is the correct font and is still using the same css style as before.
NB. The issue does not occur in Firefox

**Comment by afc163** [comment]
You should import Montserrat as webfont from https://fonts.google.com/specimen/Montserrat?selection.family=Montserrat&preview.text=127&preview.text_type=custom

**Comment by jamesdwork** [comment]
Please note my comments at the end, where I mentioned the site was previously displaying Montserrat font (ie. it was already being imported and loaded), but since we added antd, it has stopped this from happening.

**Comment by afc163** [comment]
https://54qkg.csb.app/ didn't load Montserrat font.

**Comment by jamesdwork** [comment]
I know, but the website I used does, though as I said it is not a public one. Why would antd stop it from being loaded?

**Comment by afc163** [comment]
I can't found problem until I see a reproduce that antd stop font load.

**Comment by jamesdwork** [comment]
FYI - tried removing
@import '~antd/dist/antd.css';
Then the numbers displayed with the Montserrat font again, so it is definitely the above css file that is causing issues.

**Comment by jamesdwork** [comment]
> I can't found problem until I see a reproduce that antd stop font load.

Found the cause. As previously mentioned, the numbers display the Montserrat font if we don't import the antd.css file. When we imported it, it no longer displayed in Montserrat (but letters did). I found that antd.css was overriding our font because of the following in numerous places in antd.css:
font-variant: tabular-nums;
font-feature-settings: "tnum";

These broke the font-family: Montserrat setting I had, for numbers.

So I have two options.
1) Do not import antd.css and instead just copy the styles from it that I need.
2) Import it, and override the antd override to restore the expected font (ie. restore the above css settings to the original default behaviour)
Ideally antd.css wouldn't have overridden my site styles in the first place, or documented this issue more clearly.
